libp2p/pubsub/pb/rpc.proto

```diff
@@ -0,0 +1,76 @@
+syntax = "proto2";
+
+package pubsub.pb;
+
+message RPC {
+	repeated SubOpts subscriptions = 1;
+	repeated Message publish = 2;
+
+	message SubOpts {
+		optional bool subscribe = 1; // subscribe or unsubcribe
+		optional string topicid = 2;
+	}
+
+	optional ControlMessage control = 3;
+}
+
+message Message {
+	optional bytes from_id = 1;
```

Comment:
I would just point out that `from` vs. `from_id` will be a critical thing to figure out for interop. This API won't work with other libp2p clients because of `from_id`. I think it's fine for now but worth keeping in mind for the future.